azure-pipelines.yml

```
@@ -7,58 +7,58 @@
 
 jobs:
 
-    - template: etc/ci/azure-posix.yml
-      parameters:
-          job_name: ubuntu20_cpython
-          image_name: ubuntu-20.04
-          python_versions: ['3.8', '3.9', '3.10', '3.11', '3.12']
-          test_suites:
-              all: venv/bin/pytest -n 2 -vvs tests
-
     - template: etc/ci/azure-posix.yml
       parameters:
           job_name: ubuntu22_cpython
           image_name: ubuntu-22.04
-          python_versions: ['3.8', '3.9', '3.10', '3.11', '3.12']
+          python_versions: ['3.9', '3.10', '3.11', '3.12', '3.13']
           test_suites:
-              all: venv/bin/pytest -n 2 -vvs tests
+              all: venv/bin/pytest -n 2 -vvs
 
     - template: etc/ci/azure-posix.yml
       parameters:
-          job_name: macos12_cpython
-          image_name: macOS-12
-          python_versions: ['3.8', '3.9', '3.10', '3.11', '3.12']
+          job_name: ubuntu24_cpython
+          image_name: ubuntu-24.04
+          python_versions: ['3.9', '3.10', '3.11', '3.12', '3.13']
           test_suites:
-              all: venv/bin/pytest -n 2 -vvs tests
+              all: venv/bin/pytest -n 2 -vvs
 
     - template: etc/ci/azure-posix.yml
       parameters:
           job_name: macos13_cpython
           image_name: macOS-13
-          python_versions: ['3.8', '3.9', '3.10', '3.11', '3.12']
+          python_versions: ['3.9', '3.10', '3.11', '3.12', '3.13']
           test_suites:
-              all: venv/bin/pytest -n 2 -vvs tests
+              all: venv/bin/pytest -n 2 -vvs
 
     - template: etc/ci/azure-posix.yml
       parameters:
-          job_name: macos14_cpython
+          job_name: macos14_cpython_arm64
           image_name: macOS-14
-          python_versions: ['3.8', '3.9', '3.10', '3.11', '3.12']
+          python_versions: ['3.9', '3.10', '3.11', '3.12', '3.13']
+          test_suites:
+              all: venv/bin/pytest -n 2 -vvs
```

```suggestion
              all: venv/bin/pytest -n 2 -vvs tests
```
Could you add this to all the azure pipeline tests? So we only run tests from the tests folder